### PR TITLE
srtp 2.3.0

### PR DESCRIPTION
**Pull request by igas**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/srtp.rb [hunk 1]
```diff
@@ -18,7 +18,6 @@ class Srtp < Formula
 
   def install
     system "./configure", "--disable-debug", "--prefix=#{prefix}"
-    system "make", "test"
     system "make", "shared_library"
     system "make", "install" # Can't go in parallel of building the dylib
     libexec.install "test/rtpw"
```
Comment (mistydemeo) [inline Formula/srtp.rb]: Need to remove this line if we're no longer doing `make test`; it looks like that build artifact no longer exists.

```suggestion
```

**Review by mistydemeo — CHANGES_REQUESTED** [review]

File: Formula/srtp.rb (hunk 1 above)
Comment (igas) [reply Formula/srtp.rb]: My question is - do we need this artifact? I dropped `make test` due to the audit.

Comment (mistydemeo) [reply Formula/srtp.rb]: It looks like we expect to use it in the test, so I'd say we probably actually should keep it (and keep `make test`).

**Comment by mistydemeo** [comment]
@BrewTestBot build this please

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

**Comment by mistydemeo** [comment]
The audit is a false positive for the `make test`; happy to merge it with this in place, given we need it.

**Pull request closed without merging**